docroot/profiles/custom/sitenow/sitenow.install

```diff
@@ -2459,3 +2459,16 @@ function sitenow_update_9062(&$sandbox) {
   }
 
 }
+
+/**
+ * Allow event content type to be replicated if it exists.
+ */
+function sitenow_update_9063() {
+  $config_factory = \Drupal::configFactory();
+  $config = $config_factory->getEditable('uiowa_core.settings');
+  $replicate_allowed = $config->get('uiowa_core.replicate_allowed');
+  $replicate_allowed[] = 'event';
+  $config
+    ->set('uiowa_core.replicate_allowed', $replicate_allowed)
+    ->save();
+}
```

Comment:
Should this be limited to just sites that have the event feature turned on?

Reply:
Good question. I originally went with the minimal slice for this thinking event feature might eventually be part of the default. Hopeful thinking I guess.

I think to properly do it the other way would entail:

- a config split check on the existing update hook
- an install hook for the event feature (ignored config)
- removing it from default config
- an uninstall hook for sites that turn off the feature.

All of which can be done, just wondering if it is worth it.

Reply:
Strictly from a standpoint of "Do we need to do this for it to work?" its not necessary. My concern is more that we are setting up config that doesn't match the actual options for some sites. This is probably very unlikely to cause any issues. But it is also a small detail that we are likely to miss when troubleshooting some future issue that comes down to the config option being overly broad. Maybe us having this conversation is enough.